import_export/admin.py

```diff
@@ -363,6 +363,8 @@ def get_export_queryset(self, request):
         # copied from django/contrib/admin/options.py
         list_display = self.get_list_display(request)
         list_display_links = self.get_list_display_links(request, list_display)
+        if self.get_actions(request):
+            list_display = ['action_checkbox'] + list(list_display)
 
```

Comment:
Looks you are modifying list_display after it has already been used in the previous line

Reply:
Look at django/contrib/admin/options.py, it's the same: https://github.com/django/django/blob/master/django/contrib/admin/options.py#L681

Reply:
I see :)